.github/workflows/build_and_test.yml

```diff
@@ -31,12 +31,12 @@ jobs:
       pgupgrade_image_name: "ghcr.io/citusdata/pgupgradetester"
       style_checker_image_name: "ghcr.io/citusdata/stylechecker"
       style_checker_tools_version: "0.8.18"
-      sql_snapshot_pg_version: "16.2"
-      image_suffix: "-v7693016"
-      pg14_version: '{ "major": "14", "full": "14.11" }'
-      pg15_version: '{ "major": "15", "full": "15.6" }'
-      pg16_version: '{ "major": "16", "full": "16.2" }'
-      upgrade_pg_versions: "14.11-15.6-16.2"
+      sql_snapshot_pg_version: "16.3"
+      image_suffix: "-dev-834ead7"
```

Comment:
needs to be updated before this PR can land

Reply:
```suggestion
      image_suffix: "-v13fd57c"
```